docs/docs/recipes/manage-users/using-admin-console.md

```diff
@@ -23,36 +25,56 @@ Support the partial search on the user properties: name, username, email, or pho
 
 Admin users could create a new account through Admin Console in the name of an end-user.
 
-1. On the "User Management" panel, click the "+ Add User" button.
-2. In the dialog, fill out the username and full name, and click the "Add User" button to confirm.
-3. After successfully adding, click the eye icon to show or hide the actual password.
-4. Copy the **username** and **initial password** to send to the end-user.
+On the "User Management" panel, click the "+ Add User" button.
```

Comment:
"User Mgmt" here is confusing.  just say sth like "in the upper right corner"

Reply:
After adding the screenshot, it will be clear enough?

Reply:
Updated. PTAL

```markdown
On the "User Management" panel, click the "+ Add User" button in the upper right corner.
```

Reply:
i was saying about remove the second "User Mgmt"